index.js

```diff
@@ -0,0 +1,18 @@
+module.exports = (api, options) => {
+  options.css.extract = false
+
+  api.chainWebpack(webpackConfig => {
+    webpackConfig
+      .devServer
+      .headers({
+        'Access-Control-Allow-Origin': '*',
+      })
+      .set('disableHostCheck', true)
+    
+    webpackConfig.optimization.delete('splitChunks')
```

Comment:
Setting the publicPath could solve this issue as well.

Reply:
My understanding is that when you use `optimization.splitChunks` that you have to add those chunks into your `index.html` file as `<script>` elements. Which is something that vue-cli does with HtmlWebpackPlugin but that we don't want to do in single-spa (since single-spa applications don't each have their own html file).

So for that reason we don't want `optimization.splitChunks` at all. If I'm misunderstanding something about optimization.splitChunks, I'd love to learn about it. I just tried it out locally in a test project and it behaved as I described here.

Reply:
if you split chunks you just have to set the public path, by lazy loading the App.vue into the bundle you can then set the public path + then load chunks with the path set/